src/components/ebay-combobox/examples/search-filtering.marko

```diff
@@ -0,0 +1,47 @@
+import countryList from './data.json';
+class {
+    onCreate() {
+        this.state = {
+            options: countryList,
+            autocompleteOption: 'none',
+            value: '',
+        };
+    }
+    async inputChange({ currentInputValue }, { el, index, selected }) {
+        this.state.autocompleteOption = 'none';
+        this.state.value = currentInputValue;
+        const options = await this.searchCountry(currentInputValue);
+        this.state.options = options;
+        this.state.autocompleteOption = 'list';
+    }
+
+    async searchCountry(currentInputValue) {
+        return new Promise((resolve) => {
+            clearTimeout(this.timeout);
+            this.timeout = setTimeout(() => {
+                const filtered = countryList
+                    .filter((country) =>
+                        country.name.toLowerCase().startsWith(currentInputValue.toLowerCase())
+                    )
+                    .slice(0, 10);
+                resolve(filtered);
+            }, 500);
```

Comment:
The country search feels a bit slow. Maybe this is the cause?

I realize this is just an example, but since it's likely to get copied almost verbatim by some developers, we may want to address some of these.

Reply:
Yeah, part of it is I wanted to make it so it shows how to deal with a delay. 
I had this example stored in my local for a while, just posted it for now.